### PR TITLE
fix($dirty): check if editor is dirty before updating view

### PR DESCRIPTION
**Pull request by MrMartiniMo**
Related Issue: #238 
thats my first PR, I hope I haven't messed it up :boom:

### CODE REVIEW

**Comment by deeg** [comment]
Thanks for putting this up. Will take a look over it tonight.

Something that helps us is putting up a plunker with the fix so it is easy for us to test.

Also if you could add a test which verifies your fix that would be great.

**Comment by MrMartiniMo** [comment]
Here is a plunker version of the fix.
http://plnkr.co/edit/As6nkFJPr5bVAd884dRl

I will try to set up a few tests, but it will take some time.

File: src/tinymce.js [hunk 1]
```diff
@@ -63,8 +63,10 @@ angular.module('ui.tinymce', [])
 	        $timeout.cancel(debouncedUpdateTimer);
 	         debouncedUpdateTimer = $timeout(function() {
               return (function(ed) {
-                ed.save();
-                updateView(ed);
+                if (!ed.isNotDirty) {
```
Comment (deeg) [inline src/tinymce.js]: I think we want [isDirty()](https://www.tinymce.com/docs/api/class/tinymce.editor/#isdirty) instead.

**Comment by wetternest** [comment]
this fix solves the problem for me, thx for sharing!

**Pull request closed without merging**